webserver/views/datasets.py

```diff
@@ -122,6 +122,19 @@ def _convert_dataset_to_csv_stringio(dataset):
     fp = StringIO.StringIO()
     writer = csv.writer(fp)
 
+    if dataset["description"] is not None:
+        #check if description is empty string
+        if dataset["description"] != "":
```

Comment:
You don't need to do this check twice here, you can do `if not dataset["descripton"]`, which will check against it being None and empty.

Reply:
Noted. I'll make this change and update this.